### ISSUE TITLE
lazy seqs are not printed out in Jupyter 5.0 

### ISSUE THREAD

**Issue by alldigital** [clojupyter/clojupyter#39]
After upgrading to Jupyter 5.0 the behavior changed:

Jupyter 4.4:
```
In  [1]: (map inc (range 10))
Out [1]: (1 2 3 4 5 6 7 8 9 10)
```
Jupyter 5.0:
```
In  [1]: (map inc (range 10))
Out [1]: clojure.lang.LazySeq@c5d38b66

```

**Comment by atkailash** [comment]
I get the same, jupyter 4.3.0, ipython 6.1.0, clojure 1.8.0, python 3.5.3.

```
In [1]: (def foo (take 10 (range)))
Out[1]: #'user/foo
In [2]: (filter even? foo)
Out[2]: clojure.lang.LazySeq@1b5d123
```

**Comment by aria42** [comment]
Hey all, I have a fix in #49 if you guys know who the right person to ping to get that merged is

**Comment by aria42** [comment]
This is now fixed

```terminal
Jupyter console 5.2.0
Clojupyters-0.1.0

In [1]: (map inc [1 2 3])
Out[1]: (2 3 4)
In [2]: 
```
